v2/pkg/scan/scan.go

```diff
@@ -511,7 +511,7 @@ func (s *Scanner) ACKPort(dstIP string, port int, timeout time.Duration) (bool,
 	}
 	defer conn.Close()
 
-	rawPort, err := freeport.GetFreePort()
+	rawPort, err := freeport.GetFreePort("0.0.0.0", freeport.TCP)
```

Comment:
We can use the following shortcut:
```go
rawPort, err := freeport.GetFreeTCPPort("")
```
empty address will attempt to listen on all interfaces (equivalent to `0.0.0.0`)